src/targets/gpu/lowering.cpp

```diff
@@ -96,9 +96,9 @@ struct miopen_apply
 
         add_extend_op("argmax");
         add_extend_op("argmin");
-        add_extend_op("elu");
+        // add_extend_op("elu");
         add_extend_op("gather");
-        add_extend_op("leaky_relu");
+        // add_extend_op("leaky_relu");
```

Comment:
These comments need to be removed.

Reply:
The ops aren't used so these can be removed. Should I remove the miopen calls in this PR?

Reply:
We could probably do that in a later PR.